tests/UnifiedSpecTests/Operation.php

```diff
@@ -358,6 +420,11 @@ private function executeForCollection(Collection $collection)
 
             case 'updateMany':
             case 'updateOne':
+                assertArrayHasKey('filter', $args);
+                assertArrayHasKey('update', $args);
+                assertInstanceOf(stdClass::class, $args['filter']);
+                assertThat($args['update'], logicalOr(new IsType('array'), new IsType('object')));
```

Comment:
This is necessary for update pipelines. We do the same when parsing args for bulkWrite requests.